### ISSUE TITLE
Random Skins

### ISSUE THREAD

**Issue by ticzz** [Zack2kl/Team-Based-Skins#1]
Hey man, it seems the "Random Skins.lua" stopped working.. when i load the script it loads without errors or something but when i buy guns its skined with the skin of the default skin changer. And when a new round starts it clear the skins, that works also fine.. but it seems to be unable to add a skin back to the skinslist and the final result is that i have no skin on my weapon, i have to reload my cfg to add skins back to aw skin changer and then the skin i choose in aw will be used.. till the list got deleted on next round again..

in short, right now the only job the script does is to delete your skins from skin changer when a new round starts.

Hopefully u can help me, give a solution and/or can fix it soon.

**Comment by Zack2kl** [comment]
should be fixed now
